Cargo.toml

```diff
@@ -9,6 +9,9 @@ documentation = "https://docs.rs/pledge/"
 keywords = ["openbsd", "binding", "security"]
 license = "MIT"
 
+[workspace]
+members = ["example-2015", "example-2018"]
+
```

Comment:
example-2018 needs to be a separate crate rather than an example target because it uses a different edition (Rust 2018), and achieving that by making it a workspace member allows it to share a Cargo.lock with pledge-variants (#9)

Reply:
example-2015 doesn’t strictly need to be a separate crate, but we might as well make it one for the sake of consistency

Reply:
Sensible :+1: